exporter/templates_test.go

```diff
@@ -52,18 +52,9 @@ func xmlCodeRootNode(codeDisplay models.CodeDisplay) *xml.ElementNode {
 }
 
 func TestReasonTemplate(t *testing.T) {
-	// do not negate reason, r2 compatable
+	// do not negate reason
 	reason := models.CodedConcept{Code: "REASON_CODE_1", CodeSystem: "2.16.840.1.113883.6.1"} // specified in cms9_26.json
 	rootNode := xmlReasonRootNode(reason, false, true)
-	assertXPath(t, rootNode, "//entryRelationship", map[string]string{"typeCode": "RSON"}, nil)
```

Comment:
Interesting. Because even though it lists them as such, those fields (while maybe not the actual values) exist and could be tested as well. That being said, the amount of field checking may have been pretty redundant anyway. Up to you, really. The only remaining check amounts to quite a bit on its own.